server/routes/editThread.ts

```diff
@@ -105,6 +105,9 @@ const editThread = async (models, req: Request, res: Response, next: NextFunctio
     if (title) {
       thread.title = title;
     }
+    if (url && thread.kind === 'link') {
+      thread.url = url;
```

Comment:
Even though we have client-side validation, we should still have server-side validation in order to prevent people from messing around with threads by hitting the backend directly.

Reply:
That is,  a server-side valid URL check?

Reply:
Yep